### DISCUSSION TITLE
Capillary pressure curve

### DISCUSSION THREAD

**Discussion by 15751831936** [PMEAL/OpenPNM#2721] [Q&A]
[Dear prof，
I followed the tutorial for a drainage simulation, but I found some problems when I customized the fluid properties. When I set the contact angle greater than 90 degrees I can get the capillary pressure curve。but when I set the contact angle less than 90 degrees, I can't get the capillary pressure curve. I tried print(mip) and found that no intrusion happened。
![test](https://user-images.githubusercontent.com/106420110/229330284-3d30e681-2fa6-4970-9b06-dc49b6512a62.png)

**Comment by likeduck** [comment]
Probably because you applied the trapping function

**Reply by 15751831936** [reply]
I tried to delete the trapping function but the result is still like this

**Reply by likeduck** [reply]
You can try to delete the set_outlet_BC at the same time. Its all my idea for this question.

**Reply by 15751831936** [reply]
Thanks for your help, I deleted them all but the problem persists, I think the problem might be the Washburn equation. A contact angle of less than 90 degrees may turn the calculated capillary pressure into a negative value, thus resulting in no invasion. I have been learning openpnm for a short time, maybe this idea is also wrong
